python/hsfs/storage_connector.py

```diff
@@ -363,6 +454,8 @@ def spark_options(self):
             if self._table_name is not None:
                 props["dbtable"] = self._table_name
             return props
+        elif self._storage_connector_type.upper() == self.ADLS:
```

Comment:
can you replace the ifs above also with the class variables, e.g. `== self.REDSHIFT` for consistency